chatbot/listenercenter/listenermanager/listenermanager.py

```diff
@@ -0,0 +1,143 @@
+from threading import Thread
+from http.server import HTTPServer
+from http.server import BaseHTTPRequestHandler
+import json
+
+# important! this is where all the listener subclasses are discovered.
+from ..listener import Listener
+from ..listeners import *
+
+g_body = ''
+g_listener = Listener()
+g_header_dict = dict()
+g_identity_dict = dict()
+
```

Comment:
some global variable stuff has to be used because of how python's httpserver is set up.